modules/roleReaction.js

```diff
@@ -0,0 +1,74 @@
+const firstMessage = require('../modules/firstMessage')
+
+module.exports = async (client) => { 
+
+    const channelId = '798083424015482880'
+    const getEmoji = emojiName => 
+        client.emojis.cache.find((emoji) => emoji.name === emojiName)
+
+    const emojis = {
+        counterstrike: 'csgo',
+        apex: 'apex',
+        dota2: 'tnydotatoes',
+        valorant: 'valorant',
+        tarkov: 'tarkov',
+        ghostbusters: 'ghostbusters',
+        gunbound: 'gunbound',
+        mmabois: 'mmabois',
+        l4dtime: 'l4dtime'
+    }
+
+    const reactions = []
+
+    //TODO: maybe add this as an embed
+    let emojiText = 'Welcome to TNY! We are chill group of homies who suck at games.\n\nYou\'ll usually catch us playing our main squeeze: Counter-Strike, but we also play other FPS games and party games!\nThese roles are focused on the many games and other hobbies we play/watch in this discord.\nPlease remember use the \'@\' symbol followed by the game role when you are looking for a party!\n\nReact to the corresponding role and you will be added to that game\'s role and will be notified with an \'@\' in the chat when anyone is looking for a party. Unreact to remove yourself from that role.\n'
+    for (const key in emojis) {
+        const emoji = getEmoji(key)
+        reactions.push(emoji)
+
+        const role = emojis[key]
+        emojiText += `${emoji} = ${role}\n`
+    }
+
+    firstMessage(client, channelId, emojiText, reactions)
+
+    const handleReaction = (reaction, user, add) => {
+        if (user.id === '729853777099751524') {
```

Comment:
what's the significance of this user? it'd be good to add a comment here

Reply:
that is bot alfred

Reply:
Alternatively, can we remove this check? Are there any issues with the bot being in all these roles?

It would allow us to remove hardcoding an id that is specific to an instance of this bot. So one less thing to maintain.